backends/ebpf/runtime/runtime.mk

```diff
@@ -0,0 +1,67 @@
+# If needed, bpf target files can be hardcoded here
+# This can be any file with the extension ".c"
+BPFOBJ=
+# Get the source name of the object to match targets
+BPFNAME=$(basename $(BPFOBJ))
+BPFDIR=$(dir $(BPFOBJ))
+INCLUDES+= -I$(dir $(BPFOBJ))
+
+# Arguments for the P4 Compiler
+P4INCLUDE=-I./p4include
+# P4 source file argument for the compiler (can be hardcoded)
+P4FILE=
+P4C=p4c-ebpf
+# the default target is test but it can be overridden
+TARGET=test
+# Extra arguments for the compiler
+P4ARGS=
+
+# Argument for the GCC compiler
+GCC ?= gcc
+SRCDIR=.
+BUILDDIR:= $(BPFDIR)build
+INCLUDES+= -I./$(SRCDIR) -include ebpf_runtime_$(TARGET).h
+# Optimization flags to save space
+CFLAGS+=-O2 -g # -Wall -Werror
+LIBS+=-lpcap
+SOURCES=$(SRCDIR)/ebpf_registry.c  $(SRCDIR)/ebpf_map.c $(BPFNAME).c
+SRC_BASE+=$(SRCDIR)/ebpf_runtime.c $(SRCDIR)/pcap_util.c $(SOURCES)
+# HDRS := $(BPFNAME).h $(SRCDIR)/ebpf_user.h $(SRCDIR)/ebpf_map.h $(SRCDIR)/ebpf_registry.h $(SRCDIR)/pcap_util.h
+OBJECTS = $(SRC_BASE:%.c=$(BUILDDIR)/%.o)
+DEPS = $(OBJECTS:.o=.d)
+
+# checks the executable and symlinks to the output
+all: $(BPFNAME).c $(BPFNAME)
+	@$(RM) -rf $(BUILDDIR)
```

Comment:
I don't think you should delete the directory in the makefile; perhaps in the testing script.